buildscripts/incremental/test.sh

```diff
@@ -41,7 +41,31 @@ else
     esac
 fi
 
+# If TEST_THREADING is set in the env, then check that Numba agrees that the
+# environment can support the requested threading.
+function check_sysinfo() {
+    cmd="import os;\
+         from numba.misc.numba_sysinfo import get_sysinfo;\
+         assert get_sysinfo()['$1 Threading'] is True, 'Threading layer $1 '\
```

Comment:
Does this really need to be an identity check?
```suggestion
         assert get_sysinfo()['$1 Threading'], 'Threading layer $1 '\
```

Reply:
I thought about this when writing it, I don't think there's any enforced return type for the output of `get_sysinfo` as it's largely informational only. My concern was that someone might patch this output to return a string like "It's working" or "It's not working", at which point a boolean assert would alias. If you feel differently am happy to change.

Reply:
fair. we can leave it as is